updating/updating-disconnected-cluster.adoc

```diff
@@ -1,23 +1,134 @@
 [id="updating-disconnected-cluster"]
-= Updating a disconnected cluster to a minor version
+= Updating a disconnected cluster
 include::modules/common-attributes.adoc[]
 :context: updating-disconnected-cluster
 
 toc::[]
 
-You can update, or upgrade, an {product-title} cluster that you installed in a
-restricted network to a minor version by using the web console.
+You can update, or upgrade a disconnected or fully air gapped {product-title} cluster using `oc` commandline.
 
 .Prerequisites
 
+* Have access to a conatiner registry in the disconnected environment to push and pull images. The container registry should be compatible with Docker registry API v2.
 * Have access to the cluster as a user with `admin` privileges.
 See xref:../authentication/using-rbac.adoc[Using RBAC to define and apply permissions].
 * Have a recent xref:../backup_and_restore/backing-up-etcd.adoc#backup-etcd[etcd backup] in case your upgrade fails and you must xref:../backup_and_restore/disaster_recovery/scenario-2-restoring-cluster-state.adoc#dr-restoring-cluster-state[restore your cluster to a previous state].
 
-//mirror module
+[IMPORTANT]
+====
+If you are upgrading from earlier release than 4.5.0, you must must use the manual method of creating configmap.
 
-include::modules/update-service-overview.adoc[leveloffset=+1]
+This is because `oc` command enhancement is not available in previous releases.
+====
 
-include::modules/understanding-upgrade-channels.adoc[leveloffset=+1]
+[id="update-airgapped-env"]
+== Updating an air gapped {product-title} cluster
+
+An air gapped environment is the one in which your cluster nodes cannot access the internet. For this reason you must mirror the images to a filesystem disconnected from that environment and then bring that host or removable media across that gap. It may also be the case that there are multiple clusters within the air gapped network. In such a case it makes sense to configure a central registry in the air gapped environment from which every cluster can pull required images for upgrade.
+
+.Procedure
+
+. Findout if there is link:https://access.redhat.com/solutions/4583231[an update path] exists between your current version and the version you want the cluster to move.
+
+. Mirror the version images to the internal container registry.
+
+** Connect the removable media to a system connected to the internet.
+** Mirror the images and configuration manifests to a directory on the removable media.
++
+----
+$ oc adm release mirror "--to-dir=${REMOVABLE_MEDIA_PATH}/mirror" quay.io/openshift-release-dev/ocp-release:4.3.0-x86_64
```

Comment:
This overlaps with [our existing mirror module][1].  But because #16894 and #17896 have stalled out on previous attempts at making that existing module generic enough to cover this case, I'm fine if we want to punt on a DRY fix and copy/edit over in a new location to address the update-specific variations.

[1]: https://github.com/openshift/openshift-docs/blame/4abd206b42d38e5bd66423d780c4c900b395e8ff/modules/installation-mirror-repository.adoc#L59

Reply:
And while I dislike using tags for this (#19266), if we do use them we probably want to use `...:{product-version}.0-x86_64` to get something that at least feels reasonable for whatever version of docs are actually building.  Or maybe we wrap this all up in a shell variable (`${OCP_RELEASE}`?) and avoid having any example tag or digest in the docs?